### PR TITLE
Chapter 09-3 09_3_Probando_un_Script_Bitcoin.md

### PR DESCRIPTION
**Pull request by javiervargas**

### CODE REVIEW

File: es/09_3_Probando_un_Script_Bitcoin.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+_Independientemente_ de los otros métodos de prueba que haya utilizado, probar un script en Testnet debería ser su prueba final antes de poner su script en Mainnet. No confíe en que su código sea correcto; no se limite a mirarlo. Ni siquiera confíe en los simuladores o depuradores que ha estado usando. Hacerlo es otra excelente manera de perder fondos en Bitcoin.
+
+## Resumen: prueba de un script de Bitcoin
+
+Debería instalar `btcdeb` como una herramienta de línea de comandos para probar sus scripts de Bitcoin. En el momento de escribir este artículo, produce resultados precisos que pueden recorrer todo el proceso de creación de scripts. También puede buscar en algunos sitios en línea para obtener una representación más visual. Cuando haya terminado, deberá ir a testnet para asegurarse de que todo funciona correctamente, antes de implementarlo de manera más general.
+
+## Que sigue?
+
+Continue con "Introducción a los Scripts de Bitcoin" con nuestro primer ejemplo de la vida real: [§9.4: Codificando una P2PKH](09_4_Codificando_una_P2PKH.md).
```
Comment (csralvall) [inline es/09_3_Probando_un_Script_Bitcoin.md:210]: `Continúe "Introduciendo Bitcoin Scripts"...`

File: es/09_3_Probando_un_Script_Bitcoin.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+```
+El uso de estos comandos puede hacer que sea más fácil ver lo que está sucediendo y dónde se encuentra.
+
+## Probar un script en línea
+
+También hay algunos simuladores web que puede utilizar para probar scripts en línea. Pueden ser superiores a una herramienta de línea de comandos al ofrecer una salida más gráfica, pero también encontramos que tienden a tener deficiencias.
+
+En el pasado, hemos intentado brindar pautas detalladas sobre el uso de sitios como [Script Playground](http://www.crmarsh.com/script-playground/) o [Bitcoin Online Script Debugger](https://bitcoin-script-debugger.visvirial.com/), pero se desactualizan y / o desaparecen demasiado rápido para mantenerse al día con ellos.
+
+Suponga que estos depuradores tienen la gran ventaja de mostrar las cosas visual y explícitamente que le dicen si un script tiene éxito (se desbloquea) o falla (permanece bloqueado). Asumimos que tienen desventajas con las firmas, donde muchos de ellos siempre regresan `true` para las pruebas de firmas o tienen mecanismos muy engorrosos para incorporarlas.
```
[the remark this answers is not in the document]
Comment (javiervargas) [reply es/09_3_Probando_un_Script_Bitcoin.md:191]: I don't think so,  Asumimos means "we" that is the original sense

[the remark this answers is not in the document]
Comment (csralvall) [reply es/09_3_Probando_un_Script_Bitcoin.md:191]: But in the first sentence of the paragraph `Assume` was used with the same meaning as the second `Assume` in the second sentence, and you translated it as third formal person. I'm referring to `Suponga` at the beginning of the paragraph. I agreed with the translation of the first sentence so I made the correction in the second.

**Review by csralvall — COMMENTED** [review]
Some further discussion needed in previous requested changes.

**Comment by javiervargas** [comment]
@csralvall What's pending here?

**Comment by csralvall** [comment]
> @csralvall What's pending here?

[Here](https://github.com/BlockchainCommons/Learning-Bitcoin-from-the-Command-Line/pull/335#discussion_r689682122). You translated the same word in two different ways with different persons. 

Original [here](https://github.com/BlockchainCommons/Learning-Bitcoin-from-the-Command-Line/blame/master/09_3_Testing_a_Bitcoin_Script.md#L191). Note that `Assume` was used both times.

**Comment by javiervargas** [comment]
Still I think it's correct.

**Comment by javiervargas** [comment]
@shannona ready to merge

**Pull request merged**